students/aminavi/session03/slicing_lab.py

```diff
@@ -0,0 +1,32 @@
+seq = [4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21]
+str = "lets do something with this"
+# seq
+def exchange_first_last(x):
+    return x[-1:]+ x[1:-1]+ x[:1]
+# print(exchange_first_last(str))
+assert exchange_first_last(seq) == [21,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,4]
+assert exchange_first_last(str) == 'sets do something with thil'
+
+def return_everyother(x):
+	return x[0::2] 
```

Comment:
Minor: No need to give 0 as the starting index, because it's the default when none is specified (and Python favors automatic defaults over explicitly specifying them).